main.go

```diff
@@ -14,12 +14,13 @@ import (
 )
 
 const (
-	pluginId       = "nfs"
+	pluginId = "nfs"
 )
+
 var (
-	socketAddress = filepath.Join("/usr/share/docker/plugins/", strings.Join([]string{pluginId, ".sock"}, ""))
-        defaultDir = filepath.Join(dkvolume.DefaultDockerRootDirectory, strings.Join([]string{"_", pluginId}, ""))
-        root       = flag.String("root", defaultDir, "NFS volumes root directory")
+	socketAddress = filepath.Join("/run/docker/plugins/", strings.Join([]string{pluginId, ".sock"}, ""))
```

Comment:
`gofmt` went to town on this file, but this line is the primary (only I think) change.